lib/extract_test_metadata.rb

```diff
@@ -82,5 +92,10 @@ def clean_leading_whitespace(multiline)
     def code_for_line(one_indexed_idx)
       filelines[one_indexed_idx - 1]
     end
+
+    def body_line_numbers
```

Comment:
This is used in several places, so please memoize

```suggestion
    memoize
    def body_line_numbers
```